azerothcore/azerothcore-wotlk
# fix(DB/creature_addon): Add missing auras to creature_addon

## Changes Proposed:
-  `Affected rows: 148`
-  Adds missing auras to 148 creatures that had empty fields in their creature_addon entry

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13351

## SOURCE:


## Tests Performed:

- Tested ingame. May need more testing for creatures 110538 (Flamebringer) and 12481 (Soulgrinder Ritual Bunny)
- 


## How to Test the Changes:


1. Test if the creatures and their related quests work.
2. See if the creatures listed share the auras of their stationary counterparts, use `.list aura`

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
